crates/driver/src/domain/competition/mod.rs

```diff
@@ -534,6 +544,24 @@ fn merge(solutions: impl Iterator<Item = Solution>, auction: &Auction) -> Vec<So
     merged
 }
 
+/// Returns the aimed time limit for bringing the solution onchain.
+fn time_limit(eth: &Ethereum, submission_deadline: BlockNo) -> Option<Duration> {
```

Comment:
This function is in the `competition` file where multiple time-limits exist (solve deadline, submission deadline). That's why I think the function name should be more explicit.

Reply:
renamed